### DISCUSSION TITLE
按照步骤生成新服务模块后，项目编译时报错

### DISCUSSION THREAD

**Discussion by QingKong0506** [zuihou/zuihou-admin-cloud#84] [02.提问]
会员版按照视频步骤，利用lamp-generator-plus-master/src/test/java/cloud/testinitproject.java文件生成新的服务模块，之后按照步骤，在pom.xml文件中添加了module，在nacos中添加了相关配置文件，在gateway配置文件中添加完成，之后编译整个项目报错'InitDatabaseOnStarted(com.tangyh.lamp.tenant.service.DatasourceInitDataSourceService)' in 'com.tangyh.lamp.tenant.context.InitDatabaseOnStarted' cannot be applied to '(com.tangyh.lamp.tenant.strategy.impl.DatasourceInitSystemStrategy)'。

截图如下：
![图片](https://user-images.githubusercontent.com/48938911/113802534-77249080-978d-11eb-931e-1ddb59b4ed4e.png)
![图片](https://user-images.githubusercontent.com/48938911/113802577-84417f80-978d-11eb-9af1-0c3a92572f27.png)
![图片](https://user-images.githubusercontent.com/48938911/113802602-928f9b80-978d-11eb-81d8-73e996035cfe.png)
![图片](https://user-images.githubusercontent.com/48938911/113802670-ae933d00-978d-11eb-9c70-57da13e293dc.png)
![图片](https://user-images.githubusercontent.com/48938911/113802759-d97d9100-978d-11eb-86bc-f9619eb9f0ab.png)

**Comment by zuihou** [comment]
看下是导包导错了，还是idea 误警告
